src/components/themed/TransactionListTop.js

```diff
@@ -193,7 +204,7 @@ export class TransactionListTopComponent extends React.PureComponent<Props, Stat
   handleStakePress = () => {
     const { currencyCode, walletId } = this.props
     if (currencyCode === 'FIO') Actions.push(FIO_STAKING_OVERVIEW, { currencyCode, walletId })
-    else Actions.push(STAKE_OPTIONS, { walletId, currencyCode, stakePolicies: this.state.stakePolicies })
+    else Actions.push(STAKE_OPTIONS, { walletId, currencyCode, stakePolicies: this.state.stakePolicies ?? [] })
```

Comment:
Changing components from Actions->navigation falls under the umbrella 'Replace Actions with navigation in Redux Actions' task. Per the description there, we would also need to test these changes separately to make sure there were no regressions to avoid repeating what happened last time we tried merging those kind of changes straight into develop.

Reply:
Oh, I thought the rule-of-thumb was to convert `Actions.*` to `navigation.*` piecemeal as we touch files. Can you confirm with William if that is correct/incorrect?